pkg/cri/sbserver/container_create_linux_test.go

```diff
@@ -766,7 +766,6 @@ func TestUserNamespace(t *testing.T) {
 			err: true,
 		},
 	} {
```

Comment:
We should consider making the description a field in the struct instead of using a `map[string]struct{...}`. That makes it more organic to use the tests description (and I guess also prevents the order of the tests to be randomized by go)

Reply:
Agree. I Will update to `[]struct{}{}`.